app/views/ops/_schedule_form_filter.html.haml

```diff
@@ -3,6 +3,18 @@
     .div
       -# %h4 {{buildLegend()}}
       %div{"ng-switch" => "", "on" => "scheduleModel.action_typ"}
+        .form-group{"ng-class" => "{'has-error': angularForm.target_id.$invalid}", "ng-show" => "automateRequest()"}
+          %label.col-md-2.control-label{"for" => "zone_id"}
+            = _("Zone")
+          .col-md-8
+            %select#zone_id{"ng-model"         => "_zone",
+                            "name"             => "zone_id",
+                            "ng-options"       => "zone as zone.description for zone in zones",
+                            "data-live-search" => "true",
+                            "ng-change"        => "",
```

Comment:
empty ng-change is surprising, and seems there's already a `$watch` on this field, so... not necessary?